modules/core/site.js

```diff
@@ -2127,7 +2127,6 @@ if(tableBody && !hm_mobile()) {
         sort: false,
         group: 'messages',
         ghostClass: 'drag_target',
-        draggable: ':not(.inline_msg)',
```

Comment:
@GedeonTS how is removing this line supposed to fix the issue? That property ensures that emails can still be read in inline mode when requested instead of triggering the drag action.

Reply:
That line is the one that breaks the feature, what do you suggest? @jacob-js

Reply:
@GedeonTS you're basically removing this fix: https://github.com/cypht-org/cypht/pull/1034
I suggest first investigating the true cause of the issue, and then we can implement a proper fix.

Reply:
After more investigation I confirm that the true cause of the issue is https://github.com/cypht-org/cypht/pull/1034 @jacob-js , I suggest to revert it  and find another alternative for it

Reply:
ok, so please work together to find another solution